app/infrastructure/cache/redis_cache.py

```
@@ -0,0 +1,230 @@
+            Cached value or None.
+        """
+        if not self.is_available() or self.redis is None:
+            return None
+        try:
+            value = await self.redis.get(key)
+            if value:
+                logger.debug("Cache HIT: %s", key)
+                return json.loads(value)
+            logger.debug("Cache MISS: %s", key)
+            return None
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)